data/Templates/Ccda/HistoryandPhysical.liquid

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Can we add some functional tests for these main template as well, just like the ones we do for HL7 v2 (ADT, ORU, OML, VXU)?

Reply:
Sure, I added them.